exec/src/main/java/com/optum/sourcehawk/exec/FlattenCommand.java

```diff
@@ -0,0 +1,74 @@
+package com.optum.sourcehawk.exec;
+
+import com.optum.sourcehawk.core.scan.FlattenResult;
+import lombok.extern.slf4j.Slf4j;
+import lombok.val;
+import picocli.CommandLine;
+
+/**
+ * CLI entry point for executing Sourcehawk flatten command
+ *
+ * @author Christian Oestreich
+ */
+@Slf4j
+@CommandLine.Command(
+        name = FlattenCommand.COMMAND_NAME,
+        aliases = {"flat", "merge", "converge"},
```

Comment:
Perhaps `fc` similar to `vc` alias with `validate-config`

Reply:
added